app/src/main/res/layout/fragment_settings_security.xml

```diff
@@ -27,43 +27,50 @@
         android:padding="16dp"
         android:text="@string/security_settings"
         android:textColor="@color/black"
-        android:textSize="18sp"
+        android:textSize="@dimen/text_size_title"
         app:layout_constraintLeft_toRightOf="@id/button_back" />
 
     <android.support.constraint.ConstraintLayout
         android:id="@+id/container_restore_seed"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="@color/white"
+        android:clickable="true"
+        android:focusable="true"
         android:foreground="?attr/selectableItemBackground"
         app:layout_constraintTop_toBottomOf="@id/text_title">
 
         <View
-            android:layout_width="0dp"
-            android:layout_height="0.5dp"
+            android:layout_width="@dimen/dp0"
+            android:layout_height="@dimen/divider_height_small"
```

Comment:
Stop moving all variables like 0, 0.5 or 1 to dimens or constants. Or describe your reasons for such code, please.